src/parser/jobs/sch/modules/Weaving.js

```diff
@@ -17,75 +16,32 @@ const WEAVING_SEVERITY = {
 }
 
 export default class Weaving extends CoreWeaving {
-	static dependencies = [
-		...CoreWeaving.dependencies,
-		'combatants',
-		'suggestions',
-	]
-	static title = t('sch.weaving.title')`Weaving Issues`
 	static displayOrder = DISPLAY_ORDER.WEAVING
 
-	constructor(...args) {
-		super(...args)
-		this.addHook('cast', {by: 'player'}, this._onScholarCast)
-	}
-
-	// Renamed so as not to overwrite parent module
-	// Just for tracking pos
-	_onScholarCast(event) {
-		const action = getDataBy(ACTIONS, 'id', event.ability.guid)
-		if (!action || !action.onGcd) {
-			return
+	// override to give SCH-relevant weave suggestions
+	// Pretty much copy-pasted from the CoreWeaving Module otherwise
+	_onComplete() {
+		// If there's been at least one gcd, run a cleanup on any remnant data
+		if (this._leadingGcdEvent) {
+			this._saveIfBad()
 		}
 
-		this._pos = this.combatants.selected.resources
-	}
-
-	// Now this, we want to overwrite
-	_onComplete() {
+		// Few triples is medium, any more is major
+		const badWeaves = this._badWeaves
 		this.suggestions.add(new TieredSuggestion({
 			icon: ACTIONS.SCH_RUIN_II.icon,
-			content: <Trans id="sch.weaving.suggestion.content">
-				<ActionLink {...ACTIONS.SCH_RUIN_II} /> may seem like a great choice for weaving,
-				but because its potency is <i>absurdly</i> low compared to <ActionLink {...ACTIONS.BROIL_III} />,
-				it is actually better to just clip your GCD with Broil than to waste your mana.
-				An exception is if you are moving - so the module below only tracks instances of Ruin 2 while not moving.
-			</Trans>,
-			why:  <Trans id="sch.weaving.suggestion.why">
-				<Plural value={this._badWeaves.length} one="# instance" other="# instances"/> instances of weaving with Ruin II while not moving.
+			content: <Trans id="sch.weaving.content">
+				Try to use <ActionLink {...ACTIONS.SCH_RUIN_II}/> and <ActionLink {...ACTIONS.BIOLYSIS}/> to weave your actions, and avoid weaving more actions than you have time for in a single GCD window.
+				Doing so will delay your next GCD, reducing possible uptime. Check the <a href="javascript:void(0);" onClick={() => this.parser.scrollTo(this.constructor.handle)}><NormalisedMessage message={this.constructor.title}/></a> module below for more detailed analysis.
 			</Trans>,
+			why: <Plural
+				id="core.weaving.why"
+				value={badWeaves.length}
+				_1="# instance of incorrect weaving"
```

Comment:
Nitpick (in the style of ack) but he mentioned to prefer `one` over `_1` since it makes more sense in the context of `other`.